app/models/post.rb

```diff
@@ -1,16 +1,19 @@
 class Post < ApplicationRecord
-  has_many :comments, foreign_key: :PostId
-  has_many :likes, foreign_key: :PostId
-  belongs_to :author, class_name: 'User', foreign_key: :AuthorId
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
   after_save :update_posts_counter
 
+  validates :title, presence: true, length: { minimum: 3, maximum: 250 }
+  validates :comments_counter, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Kindly make sure both `likes_counter` and `comments_counter` are of type `integer` and not `numericality` as required for the project.

Reply:
Kindly refer to this guide you will find the syntax i wrote is correct [link section 2.9](https://guides.rubyonrails.org/active_record_validations.html#validations-overview)

Reply:
Hi @mohamed20163858 

You are true